packages/core/content-releases/shared/contracts/settings.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Used to store user configurations related to releases.
+ * E.g the default timezone for the release schedule.
+ */
+
+import { errors } from '@strapi/utils';
+
+export interface Settings {
+  defaultTimezone: string | null;
+}
+
+/**
+ * GET /content-releases/settings
+ *
+ * Return the stored settings. If not set,
+ * it will return an object with null values
+ */
+export declare namespace GetSettings {
+  export interface Request {
+    query?: {};
+  }
+
+  export interface Response {
+    data: Settings;
+  }
+}
+
+/**
+ * PUT /content-releases/settings
+ *
+ * Update the stored settings
+ */
+export declare namespace UpdateSettings {
+  export interface Request {
+    body: Settings;
+  }
+
+  export interface Response {
+    data: Settings;
+    error?: errors.ApplicationError | errors.ValidationError;
```

Comment:
isn't it possible to tie the two together? To use a union to let TS know that we either have data and no error, or an error and no data.

Reply:
sure :) good point, will create a util for this!

Reply:
created an utility to make it easier on the other scenarios? wdyt? 
We can use this everywhere else from now on !